OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -93,6 +93,11 @@ public override Activity Tick(Actor self)
 				if (self.IsDisabled())
 					return this;
 
+				// If target moves inside tunnel, don't follow
```

Comment:
Do we really want this behaviour?  I imagine that ideally we would follow them through the tunnel.